src/chatkit.ts

```diff
@@ -555,6 +565,65 @@ export default class Chatkit {
     })
   }
 
+  sendMessage(options: SendMessageOptions): Promise<any> {
+    if (typeof options.sender_id === typeof undefined) {
+        throw new Error('You must provide the ID of the user that you want to set as the sender of the message');
+    }
+    if (typeof options.room_id === typeof undefined) {
+        throw new Error('You must provide the ID of the room that you want to add the message to');
+    }
+    if (typeof options.text === typeof undefined) {
+        throw new Error('You must provide some text for the message');
+    }
+    const user_id = options.sender_id;
+    const room_id = options.room_id;
+
+    interface Body {
+      text: string;
+      attachment?: {
+        resource_link: string;
+        type: 'image' | 'video' | 'audio' | 'file'
+      }
+    }
+    const body: Body = {
+      text: options.text
+    };
+
+    if (typeof options.attachment !== typeof undefined) {
+      if (!options.attachment || typeof options.attachment.resource_link === typeof undefined) {
+        console.error(options.attachment);
+        throw new Error('You must provide the resource_link for the attachment');
+      }
+      if (!options.attachment || typeof options.attachment.type === typeof undefined
+        || !(['image', 'video', 'audio', 'file'].filter(t => options.attachment && t === options.attachment.type).length ? true : false) ) {
+        throw new Error('You must provide the type for the attachment. This can be one of image, video, audio or file');
+      }
+      if (options.attachment) {
+        body.attachment = {
+          resource_link: options.attachment.resource_link,
+          type: options.attachment.type
+        };
+      }
+    }
+
+    const jwt = this.generateAccessToken({
+      su: true,
```

Comment:
This should be removed I think, or at least made an option. The default should be that it's not a `su` token so that if the user with the user ID provided in the token doesn't have permissions to send a message then the request will be rejected. This could be overridden, perhaps, as an option in the request options by letting a user set `su` to true, but it shouldn't be the default 👌